cloudferrylib/os/network/neutron.py

```diff
@@ -950,9 +950,12 @@ def allocate_floatingips(self, ext_net_id):
             while True:
                 self.neutron_client.create_floatingip({
                     'floatingip': {'floating_network_id': ext_net_id}})
-        except IpAddressGenerationFailureClient:
-            LOG.info("| Floating IPs "
-                     "were allocated in network %s" % ext_net_id)
+        except neutron_exc.NeutronClientException as e:
+            if e.status_code == 409:  # 409 - Conflict
+                LOG.info("| Floating IPs were allocated in network %s" %
+                         ext_net_id)
+            else:
+                raise
```

Comment:
Here are examples of this error on different client versions:
1. `IpAddressGenerationFailureClient: No more IP addresses available on network 8c6a3a43-e4a8-4e77-825b-544a080e6a7d.`
2. `NeutronClientException: 409-{u'QuantumError': u'No more IP addresses available on network d4e2bdf3-89bd-46f9-a284-66fc00a2b2f7.'}`
3. `NeutronClientException: 409-{u'NeutronError': {u'message': u"Quota exceeded for resources: ['floatingip']", u'type': u'OverQuota', u'detail': u''}}`

The one same thing they use - it is 409 status code)